psi-ms.obo

```diff
@@ -21888,25 +21898,162 @@ name: Lipid shorthand identification confidence - Complete structure
 def: "Lipid shorthand identification confidence level 'Complete structure'." [PMID:33037133]
 is_a: MS:1003310 ! Lipid shorthand identification confidence level
 
-[Term] 
-id: MS:1003327 
-name: number of identified protein groups 
-def: "The number of protein groups that pass the threshold to be considered identified with sufficient confidence." [PSI:PI]  
-is_a: MS:1002405 ! protein group-level result list attribute 
-is_a: MS:4000003 ! single value 
-relationship: has_metric_category MS:4000008 ! ID based metric 
-relationship: has_value_type xsd:int ! The allowed value-type for this CV term 
-relationship: has_units UO:0000189 ! count unit 
-
-[Term] 
-id: MS:1003328 
-name: number of identified proteoforms 
-def: "The number of proteoforms that pass the threshold to be considered identified with sufficient confidence." [PSI:PI] 
+[Term]
+id: MS:1003327
+name: number of identified protein groups
+def: "The number of protein groups that pass the threshold to be considered identified with sufficient confidence." [PSI:PI]
+is_a: MS:1002405 ! protein group-level result list attribute
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_value_type xsd:int ! The allowed value-type for this CV term
+relationship: has_units UO:0000189 ! count unit
+
+[Term]
+id: MS:1003328
+name: number of identified proteoforms
+def: "The number of proteoforms that pass the threshold to be considered identified with sufficient confidence." [PSI:PI]
 is_a: MS:1001085 ! protein-level identification attribute
 is_a: MS:4000003 ! single value
-relationship: has_metric_category MS:4000008 ! ID based metric 
-relationship: has_value_type xsd:int ! The allowed value-type for this CV term 
-relationship: has_units UO:0000189 ! count unit 
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_value_type xsd:int ! The allowed value-type for this CV term
+relationship: has_units UO:0000189 ! count unit
+
+[Term]
+id: MS:1003329
+name: loop-link spectrum identification item
```

Comment:
If we're scrubbing out the hyphens in crosslink, should they also be scrubbed from looplink?

Reply:
well, this is a new term so its not being "scrubbed". 
But yes, Alexander Lietner proposed that is inconsistent to to have "loop-link" hyphenated when crosslink is not.
But then Lutz and I decided to have it hyphenated still, for no better reason than we felt it looked too weird without the hyphen. 
So, the hyphen currently present in "loop-link" is deliberate but needs further discussion. I can see Eric's/Alexander's point.